thirdparty/CMakeLists.txt

```diff
@@ -1,3 +1,6 @@
+find_package(yaml-cpp)
+find_package(span-lite)
```

Comment:
Could you add a note here saying `# TODO look into using FetchContent's FIND_PACKAGE_ARGS when next increasing CMake version`

Reply:
This is done